app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def create_file_line():
+    file_name = input("Please, enter the file name: ")
+
+    while True:
+        line = input("Please, enter new line of content: ")
+        if line == "stop":
+            break
+        else:
+            with open(file_name + ".txt", "a") as f:
```

Comment:
It's better to open the file once before the loop.
Opening the file for every line can take some time.